test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,48 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const path = 'https://api.github.com/user/following/aperdomob';
```

Comment:
why not leave this without the username, so you don't have to do a replace in the second test.

Reply:
resolved